lua/nvim-treesitter/fold.lua

```diff
@@ -30,9 +30,11 @@ local folds_levels = tsutils.memoize_by_buf_tick(function(bufnr)
       stop = stop + 1
     end
 
+    local should_fold = start + 1 > stop -- Only fold for 2+ lines
```

Comment:
I came here from https://github.com/nvim-treesitter/nvim-treesitter/issues/1226 trying to see what's wrong. This should probably be `start + 1 < stop`, should it not? I'll try it out locally and see it that works

Reply:
Lol yeah, that's definitely it

Reply:
Yup, that's it :smile: I've raised https://github.com/nvim-treesitter/nvim-treesitter/pull/1234 with the fix